ui/src/app/views/recent-work/component.ts

```diff
@@ -14,21 +23,57 @@ import {
 export class RecentWorkComponent implements OnInit {
   resourceList: RecentResource[];
   fullList: RecentResource[] = [];
+  @Input('workspace')
+  workspace: Workspace;
+  accessLevel: WorkspaceAccessLevel;
   startIndex = 0;
+  cssClass: string;
+  resourcesLoading = false;
   constructor(
-    private userMetricsService: UserMetricsService
+    private userMetricsService: UserMetricsService,
+    private workspacesService: WorkspacesService,
+    private cohortsService: CohortsService,
+    private route: ActivatedRoute,
   ) {}
   index: Number;
   size = 3;
+
   ngOnInit(): void {
+    this.resourcesLoading = true;
     this.updateList();
   }
 
   updateList(): void {
-    this.userMetricsService.getUserRecentResources().subscribe((resources) => {
-      this.fullList = resources;
-      this.resourceList = this.fullList.slice(this.startIndex, this.startIndex + this.size);
-    });
+    if (this.workspace) {
+      this.cssClass = 'for-list';
```

Comment:
I still don't think this style divergence is intentional in the mocks or necessary, do you mind filing a ticket to track?

Reply:
Sure, I'm going to tie it to the other tickets that I think will resolve this as a side effect (resizing issues, redux of homepage), so that we can resolve it immediately if I'm right...